### PR TITLE
Emoji Optimizations for Mobile Users

### PR DESCRIPTION
**Pull request by mattdax**

### CODE REVIEW

File: client/sass/main.sass [hunk 1]
```diff
@@ -10,16 +10,10 @@ body
...
   overflow-y: hidden
 
-footer
-  bottom: 0
-  padding: 10px
-  position: absolute
-  width: 100%
-
-  a
+a
```
Comment (nikitavoloboev) [inline client/sass/main.sass]: what is this a here?

Comment (mattdax) [inline client/sass/main.sass]: It was there in the footer section, I just took it along with the other code any moved it up to body. Idk but it seemed to work on my phone

**Comment by nglgzz** [comment]
The problem is still there :disappointed: 
mobile-issue-with-emoji-reference

**Comment by nglgzz** [comment]
After the contribution button, I decided to just hide the other footer buttons on mobile. 
This way we don't have this problem anymore, the contribution button gets more space, and everything looks cleaner.

**Pull request closed without merging**